pawno/include/modules/src/teaminfotd.inc

```diff
@@ -0,0 +1,259 @@
+/*
+  
+    Team information on the right corner
+  
+*/
+
+#include "modules\header\teaminfotd.txt"
+
+CreateTeamCornerTextDraws()
+{
+    // Attackers textdraws
+	TeamCornerInfoFirstTD[0] = TextDrawCreate(632.000000, 360.000000, "_");
+	TextDrawAlignment(TeamCornerInfoFirstTD[0], 3);
+	TextDrawBackgroundColor(TeamCornerInfoFirstTD[0], 0x00000050);
+	TextDrawFont(TeamCornerInfoFirstTD[0], 1);
+	TextDrawLetterSize(TeamCornerInfoFirstTD[0], 0.180000, 1.000000);
+	TextDrawColor(TeamCornerInfoFirstTD[0], -1);
+	TextDrawSetOutline(TeamCornerInfoFirstTD[0], 1);
+	TextDrawSetProportional(TeamCornerInfoFirstTD[0], 1);
+	
+	TeamCornerInfoSecondTD[0] = TextDrawCreate(632.000000, 324.000000, "_");
+	TextDrawAlignment(TeamCornerInfoSecondTD[0], 3);
+	TextDrawBackgroundColor(TeamCornerInfoSecondTD[0], 0x00000050);
+	TextDrawFont(TeamCornerInfoSecondTD[0], 1);
+	TextDrawLetterSize(TeamCornerInfoSecondTD[0], 0.180000, 1.000000);
+	TextDrawColor(TeamCornerInfoSecondTD[0], -1);
+	TextDrawSetOutline(TeamCornerInfoSecondTD[0], 1);
+	TextDrawSetProportional(TeamCornerInfoSecondTD[0], 1);
+	
+	// Defenders textdraws
+	TeamCornerInfoFirstTD[1] = TextDrawCreate(632.000000, 360.000000, "_");
+	TextDrawAlignment(TeamCornerInfoFirstTD[1], 3);
+	TextDrawBackgroundColor(TeamCornerInfoFirstTD[1], 0x00000050);
+	TextDrawFont(TeamCornerInfoFirstTD[1], 1);
+	TextDrawLetterSize(TeamCornerInfoFirstTD[1], 0.180000, 1.000000);
+	TextDrawColor(TeamCornerInfoFirstTD[1], -1);
+	TextDrawSetOutline(TeamCornerInfoFirstTD[1], 1);
+	TextDrawSetProportional(TeamCornerInfoFirstTD[1], 1);
+	
+	TeamCornerInfoSecondTD[1] = TextDrawCreate(632.000000, 324.000000, "_");
+	TextDrawAlignment(TeamCornerInfoSecondTD[1], 3);
+	TextDrawBackgroundColor(TeamCornerInfoSecondTD[1], 0x00000050);
+	TextDrawFont(TeamCornerInfoSecondTD[1], 1);
+	TextDrawLetterSize(TeamCornerInfoSecondTD[1], 0.180000, 1.000000);
+	TextDrawColor(TeamCornerInfoSecondTD[1], -1);
+	TextDrawSetOutline(TeamCornerInfoSecondTD[1], 1);
+	TextDrawSetProportional(TeamCornerInfoSecondTD[1], 1);
+    return 1;
+}
+
+ShowTeamCornerInfo(playerid)
+{
+	if(ShowHPBars)
+		return 0;
+		
+	switch(Player[playerid][Team])
+	{
+	    case ATTACKER:
+		{
+		    TextDrawShowForPlayer(playerid,TeamCornerInfoFirstTD[0]);
+			TextDrawShowForPlayer(playerid,TeamCornerInfoSecondTD[0]);
+			
+			TextDrawHideForPlayer(playerid,TeamCornerInfoFirstTD[1]);
+			TextDrawHideForPlayer(playerid,TeamCornerInfoSecondTD[1]);
+		}
+		case DEFENDER:
+		{
+		    TextDrawShowForPlayer(playerid,TeamCornerInfoFirstTD[1]);
+			TextDrawShowForPlayer(playerid,TeamCornerInfoSecondTD[1]);
+			
+			TextDrawHideForPlayer(playerid,TeamCornerInfoFirstTD[0]);
+			TextDrawHideForPlayer(playerid,TeamCornerInfoSecondTD[0]);
+		}
+	}
+    return 1;
+}
+
+UpdateTeamCornerInfo()
+{
+	if(ShowHPBars)
+		return 0;
+	
+    // Variables for sorting	
```

Comment:
So man I've always wondered why you and whoever wrote the same thing for attdef liked sorting these info so much? Do they really need to be sorted in this place? I mean, you already get them sorted in match results at the end of the round and I never needed to see them sorted while PLAYING the round. The fact that this function is called very often and you have to do all that processing over and over really gets on my nerves 😞